objects/ship/avalitechstation/avalitechstation.object

```diff
@@ -35,8 +37,7 @@
 
     }
   ],
-  
-  "turnInQuests" : [ "gaterepair" ],
+
   "offeredQuests" : [ "gaterepair" ],
```

Comment:
I _assumed_ it was. The vanilla techstation.object files for all the other races don't have that line, and since their SAIL wakeUp dialog works fine, it seemed logical to put two and two together. But, again, even after that code line was taken out, the bug still occurs, so I don't know. That's why I went ahead and added the bug as an issue.